CHANGELOG

```diff
@@ -1,6 +1,20 @@
 devel
 -----
 
+* Fixed reloading of users within the Cluster
+	If a Coordinator is asked to reload it's users (e.g. by the UserManager in
+	Foxx, it is also possible to do via API, but this is internal and on purpose
+	not documented, so unlikely that you hit this), in concurrency with user
+	management updates there is a chance that the reload is not correctly performed
+	on this coordinator. It may have missed the last update locally, causing
+	one user to have an older state. It will be fixed on the next modification
+	on any other users rights. Unfortunately this bug can cascade, if you are
+	in a bug situation, and hit the same bug again, this coordinator is now
+	off by two updates.
+	In DC-2-DC this situation is more likely to happen on the follower
+	datacenter, causing this datacenter to seemingly having other rights
+	then the main one.
```

Comment:
```suggestion
* Fixed reloading of users within the Cluster.
  If a Coordinator is asked to reload its users (e.g. by the UserManager in
  Foxx, it is also possible to do via API, but this is internal and on purpose
  not documented, so unlikely that it is used), in concurrency with user
  management updates there is a chance that the reload is not correctly performed
  on this coordinator. It may have missed the last update locally, causing
  one user to have an older state. It will be fixed on the next modification
  of any other users/permissions. Unfortunately this bug can cascade and when
  hit again, the coordinator can now be off by two updates.
  In DC2DC this situation is more likely to happen on the target datacenter, 
  causing this datacenter to have other users/permissions than the source one.
```